styles.css

```diff
@@ -1,3 +1,3 @@
-.main  {
-    color: black;
-}
\ No newline at end of file
+.main {
+  color: #000;
+}
```

Comment:
### GENERAL REVIEW

- Hey, I understand that you restarted the module and have submitted a PR that has already been approved. Although it has been approved, however, you've not met all the project requirements. see them below:
   - The readme template has been changed. kindly update the readme using this [template](https://github.com/microverseinc/readme-template), then customize it to suit your project needs. Also, remove the placeholder text as they're there to guide you on how to add the content you want to in making the readme more professional.
   - The linters file is incomplete as you've only added the `.github/workflow`, you still need to add the `.hintrc` and `.stylelintrc.json`. kindly use this [html-css](https://github.com/microverseinc/linters-config/tree/master/html-css) to add the remaining.


### WHAT YOU SHOULD DO 

- Since this PR is already merged, kindly create another PR, after you've made the changes I've suggested, paste the link to this already merged PR in the newly created Pull Request Description. [Paste this](https://github.com/mugishayannick/Hello-project-Module-1/pull/5/files#diff-380b7b38760dd442e897eb0164c58f6a17da966ccaca6318017a468c163979b1L1-R3). I will leave a note on this PR to other CR to proceed the preview on the new PR.
- Then also paste the link to the newly created PR here in this already merged PR description so they can access the newly created PR's link from here. 
- Finally, resubmit this same already merged PR link when you want to ask for another review.
:point_down: 
***NOTE: please, you should submit this same PR when you want to request another review!***